app/routes/docs/producers.mdx

```diff
@@ -58,13 +58,36 @@ also happy to work with the mission teams to help construct your alerts.
 
       ```python
       from gcn_kafka import Producer
+      import json
       # Connect as a producer.
       # Warning: don't share the client secret with others.
       producer = Producer(client_id='fill me in', client_secret='fill me in')
       # any topic starting with 'mission.'
       topic = 'mission.example'
-      data = b'...'  # any bytes
-      producer.produce(topic, data)
+
+      #Construct a JSON structured notice that follows the GCN Universal Schema, and references your notice's specific schema.
+
+      example_notice =
+      {'$schema': 'https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json',
```

Comment:
This will cause a syntax error. To fix, put the opening brace on the same line as the assignment.

```suggestion
      example_notice = {
          '$schema': 'https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json',
```